test/integration_test/basic_test.go

```diff
@@ -492,7 +492,7 @@ func BasicTelemetryRegression(tc *types.TestCase) func(*testing.T) {
 		if cluster.Spec.Monitoring != nil {
 			if cluster.Spec.Monitoring.Telemetry != nil {
 				if cluster.Spec.Monitoring.Telemetry.Enabled {
```

Comment:
nit: we can remove this condition check

Reply:
I will leave it for now as we are validating that its not enabled by default

Reply:
yes, but if it's enabled, it still return error. it's a duplicated condition check. but I'm ok with that.

Reply:
I misunderstood what you said, done ^^